### DISCUSSION TITLE
How to use sahi in mmdetection

### DISCUSSION THREAD

**Discussion by yanrihong** [obss/sahi#768] [Q&A] [answered]
hi, i wander how to use sahi inmmdetection to train large picture (not in colab)
have a good day!

**Comment by fcakyon** [comment, marked as answer]
You can take these mmdet config files as reference for sliced fine-tuning: https://github.com/fcakyon/small-object-detection-benchmark/tree/main/mmdet_configs

Or you can use `slice_coco` method to slice your dataset and perform raining on any framework: https://github.com/obss/sahi/blob/main/docs/slicing.md

**Reply by yanrihong** [reply]
thanks! i m rushing to check it out

**Reply by zhangjinrong731** [reply]
Hello, I would like to know how to restore the detection results to a larger image in the mmdetection code after cropping the original dataset using slice_coco
